README.md

```diff
@@ -6,8 +6,14 @@ Highcharts component for react.
 * see [example of using react-highcharts](http://kirjs.github.io/react-highcharts/)
 * see [example of using react-highcharts/more](http://kirjs.github.io/react-highcharts/more.html)
 
-You can find the full code for the examples [here](https://github.com/kirjs/react-highcharts/tree/master/demo
-)
+You can find the full code for the examples [here](https://github.com/kirjs/react-highcharts/tree/master/demo)
+
+To run the demo, clone the repo, then run:
+```
+npm install
+npm run demo
```

Comment:
Once you run the demo, it shows the address for a second, but then webpack build kicks in, and it gets lost, worth mentioning in the readme that the user should open `http://localhost:8080/webpack-dev-server/` or maybe just opening it in the browser right away (with open package, or any other way)